app/utils/paths.ts

```diff
@@ -18,6 +18,7 @@
 
 import AppConfig from '../config';
 import { TS } from '-/tagspaces.namespace';
+import PlatformIO from '-/services/platform-io';
```

Comment:
this file should not have dependencies to PlatformIO, this could cause issue with unit tests